Fullcontact

MrJester opened the pull request:
Removed the old one and brushed up this code to properly parse and import contacts. Also removed the hardcoded API key.... :)

lanmaster53 commented:
Was this broken? I just used this module the other day and it seemed to work great. Does the new API give us anything new that the old one didn't? Is it being deprecated soon or something? Just curious on what prompted the change.

MrJester commented:
It wasn't working for me. There was also a fullcontact-new module that had
a key in the source code. So I removed the fullcontact module and replaced
it with this new one removing the key and making the tweaks to get it to
work.

On Mon, Jun 17, 2019 at 10:49 AM lanmaster53 <notifications@github.com>
wrote:

> Was this broken? I just used this module the other day and it seemed to
> work great. Does the new API give us anything new that the old one didn't?
> Is it being deprecated soon or something? Just curious on what prompted the
> change.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lanmaster53/recon-ng-modules/pull/29?email_source=notifications&email_token=ABCS7ZRTD4D2S62RRK67DUDP26P6ZA5CNFSM4HX2NVGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODX3NGSQ#issuecomment-502715210>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCS7ZXH3VCTYTOVIWIJLU3P26P6ZANCNFSM4HX2NVGA>
> .
>


-- 
Ryan Hays
757-869-3428

When someone says, "If evolution is real, why are there still monkeys?"

Tell them, "Education is real and there are still morons.

lanmaster53 commented:
Uh... ok. That wasn't supposed to be in there. Lol. Somehow I committed some dev code into the module repo. It's not in the old Recon-ng repo where the modules are embedded. I hope that was the only one. I rotated that API key, so I'm not concerned about it being in previous commits.

lanmaster53 commented:
Bah! I merged without realizing you forgot to update the indexes! Can you do that real quick and submit a pull request for that?

lanmaster53 commented:
Never mind. I got it.
